apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -3610,7 +3393,7 @@
 <e>       <p><l>ekstremist<s n="n"/></l>               <r>ekstremist<s n="n"/></r></p></e>
 <e>       <p><l>ekvator<s n="n"/></l>                  <r>ekvator<s n="n"/></r></p></e>
 <e>       <p><l>ekvivalent<s n="n"/></l>               <r>ekvivalent<s n="n"/></r></p></e>
-<e>       <p><l>el<s n="n"/></l>                       <r>&quot;el<s n="n"/></r></p></e>
+<e>       <p><l>el<s n="n"/></l>                       <r>el<s n="n"/></r></p></e>
```

Comment:
Why are there two sections in the dictionary both in the same place in the alphabet?  This is the underlying cause for duplicates.

Reply:
I thought  there is no need to quot so I removed it. I  don't understand your question, Could you clarify more about sections? You mean sections common and GSoC2019 in the dictionary?

Reply:
most of the words in section common are [uzb-uzb] and in section GsoC2019 right translation are existing. The words which I removed are existing in section GSoC2019 with translation.

Reply:
My question is why are there two sections that have `<l>ek`..., `<l>el`..., etc.?  I mean, the entire dictionary should be alphabetically sorted, but it seems like there are two sections.

Reply:
> right translation are existing

As I said before, **all** translations, even if extremely rare, should exist in the dictionary.  A single word can (and often does) have more than one translation in another language!

Reply:
@IlnarSelimcan, could you clarify what's going on with sections?

Reply:
Yes, I mean entire dictionary divided in two sections. Both sections alphabetically sorted. the section GSoC2019 added after the section common.  I think .dix file should be sorted again and we should remove true duplicates.

Reply:
Yes, I agree.  There's a tool for this: http://wiki.apertium.org/wiki/Apertium-dixtools

Let me know how it goes!

Reply:
Note that sorting the dictionary and fixing individual entries should probably be separate commits so that it's easier to track what was changed in the fixes.